util/Migrator/DbMigrator.cs

```diff
@@ -56,17 +51,14 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
         return false;
     }
 
-    public bool MigrateDatabase(bool enableLogging = true,
-        bool repeatable = false,
-        string folderName = MigratorConstants.DefaultMigrationsFolderName,
-        CancellationToken cancellationToken = default(CancellationToken))
+    private void PrepareDatabase(CancellationToken cancellationToken = default)
     {
-        if (_logger != null)
```

Comment:
ℹ️ This is now lower down.

❓ There was a confusing mix of logger usages in this class so I just went with "use it or not". The only project not logging is `Setup` -- why? Do we want to suppress setup log messages?